### ISSUE TITLE
WSL doesn't need the @cjknarrow locale modifier

### ISSUE THREAD

**Issue by goreliu** [mintty/mintty#686]
When I set mintty Locale+Charset to zh_CN.UTF-8, and pass LANG to wslbridge, @cjknarrow will be appended to LANG. But WSL doesn't need it.

```
> mintty --wsl -o Locale=zh_CN -o Charset=UTF-8 /bin/wslbridge -e LANG bash -c 'echo $LANG; cat'
zh_CN.UTF-8@cjknarrow
```

**Comment by mintty** [comment]
In that case, which I wasn't aware of, mintty does in fact still make its locale setting dependent of the font properties. If you also set a "wide" font, this should not happen.
Since WSL does not know the @cjknarrow locales, this derivation should be dropped at least for `--wsl`, or maybe at all?

**Comment by goreliu** [comment]
I think disable the modifier for `--wsl` will work well.

**Comment by mintty** [comment]
Released 2.8.0.
